README.md

```diff
@@ -301,6 +301,25 @@ In particular:
 
 
 
+#### Github using SSH
+To use terragrunt with SSH github authentication, use the following syntax. Note the `/` after 
+github.com instead of `:` and the uses of `//`. 
+
+```hcl
+terragrunt = {
+  terraform {
+    source = "git::ssh://git@github.com/foo/modules.git//path/to/module?ref=v0.0.1"
+  }
+}
+```
+For private git repositories, replace github.com with the private repository host. 
```

Comment:
@brikis98 I think I should leave in the comment about `:` as the point of this extra documentation is  partially that using it does not work after the host and your bitbucket example here included it ;). Past that will update later today.